### ISSUE TITLE
Missing warning before sending to many recipients

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/mail#6461]
### Steps to reproduce

1. Send a message to 11 people

### Expected behavior

The app warns that the message will go to lots of recipients thanks to https://github.com/nextcloud/mail/pull/4838.

### Actual behavior

The app just sends without confirmation.

### Mail app version

v1.12.0

**Comment by miaulalala** [comment]
Since the send action is divorced from the message creation, at which point should the error be displayed?

**Comment by ChristophWurst** [comment]
For better UX I'd say when the message is added to the outbox

**Comment by ChristophWurst** [comment]
Label dropped because it was a duplicate of the attribute at https://github.com/orgs/nextcloud/projects/61. The priority does **not** change.

**Comment by miaulalala** [comment]
Blocked by #7517
